conference/submissions.md

```diff
@@ -4,7 +4,51 @@ layout: default
 
 # Submit a proposal
 
-We welcome any type of proposal for this conference.  This is the
+
+### COVID-19 statement
+
+The organisers hope to be able to proceed with running
+the Nordic-RSE conference, as planned, in Stockholm on December 1-2.
+However, with the ongoing COVID-19 situation, we are aware that this relies on
+there being a continuing improvement in the situation over the coming months.
+It is also likely that social distancing measures will have some effect on our
+plans for the conference and full details will be advised nearer to the
+conference. We encourage you to go ahead with making a submission to the
+conference but urge you not to make any travel bookings that are non-refundable
+at this stage ([COVID-19 decision timeline](/conference/timeline/)).
```

Comment:
Maybe move this slightly lower?  The next section is a good introduction and should probably be first.

Reply:
Done